packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -32,25 +29,32 @@ interface ICommonProps {
 	completedGCRuns: number;
 	isTombstoned: boolean;
 	lastSummaryTime?: number;
-	fromId?: string;
 	viaHandle?: boolean;
 }
 
 /** The event that is logged when unreferenced node is used after a certain time. */
 interface IUnreferencedEventProps extends ICreateContainerMetadata, ICommonProps {
 	state: UnreferencedState;
-	id: string;
+	id: {
+		value: string;
+		tag: string;
+	};
 	type: GCNodeType;
 	unrefTime: number;
 	age: number;
 	timeout?: number;
+	fromId?: {
+		value: string;
+		tag: string;
+	};
 }
 
 /** Properties passed to nodeUsed function when a node is used. */
 interface INodeUsageProps extends ICommonProps {
 	nodeId: string;
 	currentReferenceTimestampMs: number | undefined;
 	packagePath: readonly string[] | undefined;
+	fromNodeId?: string;
```

Comment:
Should `IUnreferencedEventProps` and `INodeUsageProps` be consistent on whether it's `id`/`fomrId` or `nodeId` and `fromNodeId`?